test/ntapfuse_test.py

```diff
@@ -0,0 +1,194 @@
+# ntapfuse unit testing version 0.1
+# code by @author Carter S. Levinson <carter.levinson@pitt.edu>
+from operator import sub
+import unittest
+import subprocess
+import sqlite3
+import regex
+import pytest
+import os
+
+# Note: that there is shell builtin command to interface with every syscall
+
+# Good tests can be thought of as operating in four steps
+# 1. Arrange: prepare the environment for our tests
+# 2. Act: the singular state changing action we want to test
+# 3. Assert: examine the resulting state for expected behaviour
+# 4. Cleanup: the test must leave up no trace
+
+# TODO: set up fixtures for temp environment and think about invocation
+# procedures
+# TODO: define clear list of what we want to be tested, what the edge cases are
+# and ideal behavior
+# TODO: integrate with sqlite 3
+# TODO: think about how the OS and the subprocess layer affect our testing
+# environment and procedure
+
+mountName = "mountPoint"
+baseDir = "testbase"
+dbName = "log.db"
+blockSize = 4096
+usage=0
+numLogs = 0
+
+# need to do more thinking about how this will work
+def setup_test_env():
+    # con = sqlite3.connect('log.db')
```

Comment:
Delete the commented code lines if they are not needed - to improve readability.

Reply:
got it